frontend/types/api/block.ts

```diff
@@ -1,6 +1,6 @@
 // Code generated by tygo. DO NOT EDIT.
 /* eslint-disable */
-import type { ApiDataResponse, Hash, Address, ClElValue } from './common'
+import type { ApiDataResponse, Hash, Address, ClElValue } from "./common";
```

Comment:
we should make an exception for the api folder as these are code generated and we should not auto-format them

Reply:
had the exception in place, but then used a `cli` command without `respecting` the `ignorePath` 🙈

Reply:
✅